src/components/ResultListItem.tsx

```diff
@@ -0,0 +1,205 @@
+  max-width: ${StyleWidths.width4};
+  min-height: 10rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.values.sm + 'px'}) {
+    display: block;
+  }
+`;
+
+const StyledFirstColumn = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: 11rem;
+`;
+
+const StyledThumbnailWrapper = styled.div`
+  background-color: ${Colors.DescriptionPageGradientColor1};
+`;
+
+const StyledThumbnailMetadata = styled.div`
+  display: flex;
+  align-items: center;
+  height: 1.5rem;
+`;
+
+const StyledFileTypeIcon = styled.span`
+  margin: 0.5rem 0.3rem 0.5rem 0.5rem;
+`;
+
+const StyledTimeCreatedTypography = styled(Typography)`
+  min-width: 6rem;
+`;
+
+const StyledMaxTwoLinesTypography = styled(Typography)`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-line-clamp: 2; /* number of lines to show. Not IE */
+  -webkit-box-orient: vertical;
+`;
+
+const StyledMaxOneLineTypography = styled(Typography)`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-line-clamp: 1; /* number of lines to show. Not IE */
+  -webkit-box-orient: vertical;
+`;
+
+// const StyledLicense = styled.div`
+//   margin-top: 1rem;
+// `;
+
+const StyledFileName = styled(Typography)`
+  max-width: 6rem;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+`;
+
+const StyledSecondColumn = styled.div`
+  padding-left: 1rem;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  min-height: 9rem; //TODO: +1rem when license
+  @media (max-width: ${({ theme }) => theme.breakpoints.values.sm + 'px'}) {
+    padding-left: 0;
+    padding-top: 1rem;
+    min-height: inherit;
+  }
+`;
+
+const StyledHeader = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 0.5rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.values.sm + 'px'}) {
+    flex-direction: column;
+  }
+`;
+
+const StyledChip = styled(Chip)`
+  margin-right: 0.5rem;
+  margin-top: 0.5rem;
+`;
+
+interface ResultListItemProps {
+  resource: Resource;
+}
+
+const ResultListItem: FC<ResultListItemProps> = ({ resource }) => {
+  const { t } = useTranslation();
+
+  const getStyledFileTypeIcon = (type: string) => {
+    if (type.toUpperCase() === ResourceFeatureTypes.audio.toUpperCase()) return <VolumeUpIcon />;
+    if (type.toUpperCase() === ResourceFeatureTypes.image.toUpperCase()) return <PhotoOutlinedIcon />;
+    if (type.toUpperCase() === ResourceFeatureTypes.presentation.toUpperCase()) return <SlideshowIcon />;
+    if (type.toUpperCase() === ResourceFeatureTypes.simulation.toUpperCase()) return <SlideshowIcon />;
+    if (type.toUpperCase() === ResourceFeatureTypes.video.toUpperCase()) return <VideocamIcon />;
+    if (type.toUpperCase() === ResourceFeatureTypes.document.toUpperCase()) return <DescriptionOutlinedIcon />;
+    return <InsertDriveFileIcon />; //default //TODO: "audiovisual" ?
```

Comment:
hva gjør vi med de typer som ikke støttes ?

Reply:
I registreringsskjemaet så har jeg satt ting som ikke uppy klarer å gjette hva er for noe til Document mener jeg.